BookPlayer/Player/Containers/PlayerControlsViewController.swift

```diff
@@ -86,8 +102,12 @@ class PlayerControlsViewController: PlayerContainerViewController, UIGestureReco
         NotificationCenter.default.addObserver(self, selector: #selector(self.onPlayback), name: Notification.Name.AudiobookPlayer.bookPlaying, object: nil)
     }
 
-    override func didReceiveMemoryWarning() {
-        super.didReceiveMemoryWarning()
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+
+        if !UserDefaults.standard.bool(forKey: UserDefaultsConstants.artworkJumpControlsUsed) {
```

Comment:
why not use `self.artworkJumpControlsUsed`? it's already intialized in `viewDidLoad`

Reply:
Doh! Forgot to refactor that one